docs/source/tutorial/spaceflights_tutorial_faqs.md

```diff
@@ -0,0 +1,59 @@
+# Spaceflights tutorial FAQs
+
+
+## How do I resolve these common errors?
```

Comment:
BIg one to include here is installing kedro but not the dataset requirement

Reply:
Please could you spell out what that would be? Is this related to when we split out datasets into a separate package? Or is it (right now) about The requirement for Kedro the framework vs `kedro[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]` ? @datajoely

Reply:
To be fair recently we've improved the error message to spit this out if you don't have something installed:

```
'api_endpoint':
Object 'SparkDataSet' cannot be loaded from 'kedro.extras.datasets.spark'. Please 
see the documentation on how to install relevant dependencies for 
kedro.extras.datasets.spark.SparkDataSet:
https://kedro.readthedocs.io/en/stable/kedro_project_setup/dependencies.html
```

The problem is actually well described on [the linked page already](https://kedro.readthedocs.io/en/stable/kedro_project_setup/dependencies.html).